.changeset/real-waves-bathe.md

```diff
@@ -0,0 +1,5 @@
+---
+"@latticexyz/create-mud": patch
+---
+
+Added `@latticexyz/explorer` and `@latticexyz/store-indexer` to all MUD templates. Updated default `worlds.json` config and world address.
```

Comment:
```suggestion
New projects created with `pnpm create mud` now include the World Explorer and SQLite indexer running as additional services.
```

Reply:
Makes perfect sense, will try to write it like that moving forward